src/components/Card/card.module.scss

```diff
@@ -5,7 +5,8 @@
   display: block;
   font-family: var(--card-font-family);
   gap: $space-m;
-  padding: $space-ml;
+  height: 260px;
```

Comment:
still of the opinion that predefined heights for cards at the library level is too prescriptive

Reply:
Maybe we'll remove in v3, for now some UI is prob depending on this.